debian/control

```diff
@@ -82,6 +82,9 @@ Replaces:
  pt-device-manager (<< 4.0.0),
 # pt-oled
  python3-pt-oled (<< 3.0.0),
+# Festival speech engine
```

Comment:
This is currently under `Replaces:` field, which is [definitely not what you want to be doing](https://www.debian.org/doc/debian-policy/ch-relationships.html#overwriting-files-and-replacing-packages-replaces).

Are you trying to make this a core dependency, an optional dependency or a dependency of the full SDK?